src/coreComponents/physicsSolvers/fluidFlow/wells/CompositionalMultiphaseWell.cpp

```diff
@@ -73,8 +73,8 @@ CompositionalMultiphaseWell::CompositionalMultiphaseWell( const string & name,
     setInputFlag( InputFlags::OPTIONAL ).
     setDescription( "Use mass formulation instead of molar" );
 
-  this->registerWrapper( viewKeyStruct::useMassFlagString(), &m_useTotalMassEquation ).
-    setApplyDefaultValue( 0 ).
+  this->registerWrapper( viewKeyStruct::useTotalMassEquationString(), &m_useTotalMassEquation ).
+    setApplyDefaultValue( 1 ).
```

Comment:
```suggestion
    setApplyDefaultValue( m_useTotalMassEquation ).
```

Reply:
I think the intent was to use all this  machinery to set the default and avoid initializing value on class construction..